src/mame/pacman/pacman.cpp

```diff
@@ -8234,6 +8274,7 @@ GAME( 1981, pacgal,     mspacman, woodpek,  mspacman, pacman_state,  empty_init,
 GAME( 1981, mspacpls,   mspacman, woodpek,  mspacman, pacman_state,  empty_init,    ROT90,  "hack",                                  "Ms. Pac-Man Plus",                                 MACHINE_SUPPORTS_SAVE )
 GAME( 1992, mschamp,    mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Zola-Puc Gal",       MACHINE_SUPPORTS_SAVE ) // Rayglo version
 GAME( 1995, mschamps,   mspacman, mschamp,  mschamp,  pacman_state,  init_mschamp,  ROT90,  "hack",                                  "Ms. Pacman Champion Edition / Super Zola-Puc Gal", MACHINE_SUPPORTS_SAVE )
+GAME( 1981, mspacpl2,   mspacman, woodpek,  mspacman, pacman_state,  init_mspacpl2, ROT90,  "hack",                                  "Miss Packman Plus",                                MACHINE_SUPPORTS_SAVE )
```

Comment:
Maybe make the short name `mspackpls` or something, reflecting the “Packman” spelling on the title screen?

Reply:
Isn't the short name limited to 8 characters? Also, it is related to mspacpls as that is a hacked romset that lifted the levels from this one...

Reply:
They’re limited to 16 characters.  We often make clones the name of the parent with a suffix appended.

Reply:
Cool, renamed to mspackpls, thanks.